prow/plugins/lgtm/lgtm.go

```diff
@@ -334,6 +349,67 @@ func checkAllowedToLGTM(
 	return true, nil
 }
 
+// returns ok = true if the TBR is allowed in this case
+// also returns an errors encountered while handling
+// may comment on the PR
+func checkAllowedToTBR(
+	rc reviewCtx,
+	skipCollaborators bool,
+	gc githubClient,
+	ownersClient repoowners.Interface,
+	log *logrus.Entry,
+) (ok bool, err error) {
+	// get fields from rc
+	commentAuthor := rc.author
+	prAuthor := rc.issueAuthor
+	number := rc.number
+	body := rc.body
+	htmlURL := rc.htmlURL
+	org := rc.repo.Owner.Login
+	repoName := rc.repo.Name
+
+	// only the Author can TBR their PR, anyone else should use LGTM
+	isPRAuthor := commentAuthor == prAuthor
+	if !isPRAuthor {
+		resp := "you cannot TBR someone else's PR."
```

Comment:
For me, `TBR` is still a pretty opaque acronym. Maybe:

```
"you can only mark your own PRs to merge without approval"
```

or similar?

Reply:
presumably they learned about TBR when they learned about the command? and it's not really marking them for merge, it's marking them as `lgtm` + `to-be-revivewed`, unless we're making assumptions about tide and how it is configured